src/symbols/Edge.tsx

```diff
@@ -485,7 +485,7 @@ export const Edge: FC<EdgeProps> = ({
   );
 
   return (
-    <group position={[0, 0, active ? 1 : 0]}>
+    <group position={[0, 0, active || isActive || isSelected ? 1 : 0]}>
```

Comment:
Can we make these variables with a informative name - its hard to follow whats going on here and below on 531.